api/rest/v2/index.md

```diff
@@ -56,49 +208,43 @@ expressions in the `property` or `expression` parameter.
 
 The following table describes symbols in the V2 API relation expressions:
 
-| Symbol | Represents |
 | ------ | ---------- |
 | `->` | An `out` arc |
 | `<-` | An `in` arc |
-| `{property:value}` | Filtering; identifies the property and associated value |
+| <code><var>PROPERTY</var>:<var>VALUE</var></code> | Filtering; identifies the property and associated value |
```

Comment:
please keep curly brackets -- it's part of the syntax

Reply:
Oops, thanks for catching. This is one of the reason for the formatting changes: in many cases, curly brackets were used to indicate a variable, so it was ambiguous.